source/arguments.rst

```diff
@@ -16,7 +16,7 @@ Arguments::
   --resize-to <integer>
                         resizes images by the largest side for opensfm. Set to
                         -1 to disable.
-			Default: 2048
+                        Default: 2048
```

Comment:
is this an intentional change in spacing?

Reply:
Totally unintentional. Sorry this is my first time using github.

Reply:
no worries. thanks for contributing, it's great to see people helping out.

Reply:
apologies, i think my initial comment sounded rude. i've had my coffee now. and actually, it looks like you aligned things better than before.

Reply:
It's OK. The truth is that I was looking into these files trying to understand the way they were written. But after I uploaded the files I realized this file was modified. I couldn't figure out how to revert the changes made and I'm just hoping someone else can solve this in the following steps.